.github/workflows/backport.yml

```diff
@@ -16,15 +16,19 @@ jobs:
   backport:
     name: Backport Pull Request
     if: >
-      secrets.GH_TOKEN_FOR_UPDATES
```

Comment:
I was half wondering if `secrets` would work in an `if` condition. Elsewhere, in other repos we can check a `var`.